CMakeLists.txt

```
@@ -352,3 +359,40 @@ if(TEST_APPS)
     endif()
   endif()
 endif()
+
+# Export targets
+install(
+  EXPORT libSRTPTargets
+  FILE libSRTPTargets.cmake
+	NAMESPACE libSRTP::
+	DESTINATION lib/cmake/libSRTP
```

This could make target files install to correct path.
```suggestion
	DESTINATION share/libSRTP
```

---

Why is this correct, the share dest folder is not used as far as I see at the moment.